truth-android-appcompat-v7/build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.library'
 
 dependencies {
-    compile project(':truth-android')
-    compile rootProject.ext.supportAppCompat
+    implementation project(':truth-android')
```

Comment:
Please change these to `api` rather than `implementation` as the dependencies are exposed as part of the library API. Please do so for all of the modules

Reply:
Fixed